### PR TITLE
Clarify working directories for patch modification

### PR DESCRIPTION
**Pull request by nullishamy**
This clarifies which repositories should be operated upon when editing existing patches, as this was previously unclear.

(Wording to be confirmed)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -99,6 +99,9 @@ Your commit will be converted into a patch that you can then PR into Paper.
 
 Modifying previous patches is a bit more complex:
 
+> ! When modifying patches, you should be operating on the sources directory
```
Comment (jpenilla) [inline CONTRIBUTING.md:102]: I feel like this is covered in the section above?

**Review by jpenilla — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (nullishamy) [reply CONTRIBUTING.md:102]: Hmm, it's stated there, but its still not clear (to me at least) that the same should be followed for modification.

**Comment by Owen1212055** [comment]
It's tricky because although it's implied I do see why there can be some confusion. Perhaps say "similarly to adding patches" or something. As the whole process of editing a patch is being in the git repo for the pserv/api, which is pretty much needed when touching the patches at all.

**Comment by Owen1212055** [comment]
Closing as in general this seems to be something that doesn't need to be clarified. Thank you for your contribution regardless.

**Pull request closed without merging**